test/lib.sh

```diff
@@ -44,6 +44,31 @@ empty_repo() {
     echo $tmpd
 }
 
+## bootstrap_project PATH TEST_PROJECT INITIAL_CONVENTION
+#
+# Build a temp boilerplated git project containing test_project files
+# - Copies in boilerplate/update from $REPO_ROOT
+# - Creates an empty boilerplate/update.cfg
+# It does not run the update.
+#
+# :param PATH: the test_project (from boilerplate/test/test_projects)
+bootstrap_project() {
+    repodir=$1
+    test_project=$2
+    (
+        cp -R $REPO_ROOT/test/test_projects/$test_project/* $repodir/.
+        pushd $repodir > /dev/null
```

Comment:
Prefer not swallowing stdio unless there's a good reason for it. E.g. when we're relying on a function to print a specific thing for the caller to consume. But even then, prefer to work around by using `>&2` so the output is still visible.

Reply:
Will update it to move forward but not sure if we should enforce blindly this logic or try to be relevant. 
The objective here is to make stdio readable/useful (avoiding to have context less repository names/list) and alligned with `cd` which is not polluting stdout.